platforms/gke/base/core/teardown.sh

```diff
@@ -78,20 +78,19 @@ for terraservice in "${terraservices[@]}"; do
     rm -rf \
       "${ACP_PLATFORM_BASE_DIR}/_shared_config/.terraform/" \
       "${ACP_PLATFORM_BASE_DIR}/_shared_config"/terraform.tfstate* \
+      "${ACP_PLATFORM_BASE_DIR}/kubernetes/kubeconfig" \
+      "${ACP_PLATFORM_BASE_DIR}/kubernetes/manifests" \
+      "${ACP_PLATFORM_CORE_DIR}/container_node_pool"/container_node_pool_*.tf \
```

Comment:
Is this necessary, now that these files are created by `container_node_pool`?

Reply:
The container_node_pool files weren't being deleted when I was testing, I can check if I had all of the latest changes pulled in.

Reply:
It looks like it was cleaned up on my recent test.